authenticity-collector/src/identity/identity.service.ts

```diff
@@ -0,0 +1,60 @@
+import { BadRequestException, Injectable, Logger } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { firstValueFrom } from 'rxjs';
+import { InjectModel } from '@nestjs/mongoose';
+import { Identity, IdentityDocument } from './schemas/identity.schema';
+import { Model } from 'mongoose';
+import { IdentityDto } from './models/IdentityDto';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class IdentityService {
+    private readonly logger = new Logger(IdentityService.name);
+
+    constructor(
+        private httpService: HttpService,
+        private configService: ConfigService,
+        @InjectModel(Identity.name) private identityModel: Model<IdentityDocument>) { }
+
+    async proveAndSaveSlu(identity: IdentityDto): Promise<Identity> {
+        this.logger.log('Proving identity');
```

Comment:
such logs are fine for development purposes but not necessary in production so please remove them and in best case remove them before creating a pr